### ISSUE TITLE
Release v1.2.3 to fix bug fixes

### ISSUE THREAD

**Issue by njwest** [trufflesuite/ganache#1059]
After I deleted a Truffle app directory that previously used Ganache for testing, Ganache crashes on application startup and says it is unable to make a "logs" dir within that now deleted project directory.

Removing and re-installing Ganache does not solve this issue 

macOS Mojave 10.14.2
PLATFORM: darwin
GANACHE VERSION: 1.2.2

EXCEPTION:
```
Error: ENOENT: no such file or directory, mkdir '/Users/user1/Desktop/my-app/logs'
    at Object.fs.mkdirSync (fs.js:891:18)
    at Object.module.exports.generateLogFilePath (/src/chain/logging.js:68:8)
    at /src/chain/chain.js:54:17
    at stopServer (/src/chain/chain.js:41:5)
    at startServer (/src/chain/chain.js:46:3)
    at process.<anonymous> (/src/chain/chain.js:158:7)
    at emitTwo (events.js:125:13)
    at process.emit (events.js:213:7)
    at emit (internal/child_process.js:768:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: The Log Directory '/Users/user1/Desktop/my-app/logs' doesn't exist; attempting to create it now
```

**Comment by seesemichaelj** [comment]
Hey @njwest, this actually got solved in https://github.com/trufflesuite/ganache/commit/505af46ba3fe2acb422285841dd59e4477193e50 which went into the `2.0.0-beta.1`, but we haven't released a `1.x` version with the fix. I'll keep this issue as a reminder to release `v1.2.3` with bug fixes

Thanks for reporting it!

**Comment by seesemichaelj** [comment]
[v1.2.3](https://github.com/trufflesuite/ganache/releases/tag/v1.2.3) was released today! This bug should be fixed in that build (as well as the 2.0 betas if you want to give those a shot)
